charts/loki-distributed/templates/networkpolicy.yaml

```
@@ -0,0 +1,190 @@
+{{- if .Values.networkPolicy.enabled }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: namespace-only-{{ template "loki.name" . }}
```

Use `include` instead of `template` everywhere to be consistent. Also, it is common practice to append suffixes to the standard fullname.
```suggestion
  name: {{ include "loki.name" . }}-namespace-only
```

---

done